spec/integration/patches/railtie_spec.rb

```diff
@@ -5,12 +5,6 @@
     cmd.stdout
   end
 
-  context "console" do
-    it "loads the console config into the selected console" do
-      expect(cmd_stdout).to include('sandbox')
```

Comment:
A unit test has been added under `context ".load_config"`, and `integration/console/irb_spec` takes care of this anyway.